proposals/0002-root-signature-in-clang.md

```diff
@@ -470,6 +471,19 @@ for Root Signature Blob or DX Container.
 
 ## Detailed design
 
+### Root Signature Versioning 
+
+Root Signature have multiple versions that require supporting. To support it,
+a new optional flag called `-force-rootsig-ver` will be added to clang dxc
```

Comment:
I think adding the new flag to the driver should be in the Proposed Solution section.  Currently that's arranged:

* AST
* LLVM IR
* DX Container Blob Generation

I think it'd be good to add a "Command-line options" or "Driver" section (not sure on exact technical term for this) before the AST section, since it's the first thing in the compilation pipeline.